src/sentry/templates/sentry/auth_close.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <script>
+  // sends message to window that opened this popup if user is authenticated and closes the window
+  function extractDomain(url) {
+    var matches = url.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i);
+    var domain = matches && matches[1];
+    return domain;
+  }
+
+    (function checkAuthChange() {
+      var logged_in = '{{ logged_in }}';
+      if (window.opener && logged_in === 'True') {
+        window.opener.postMessage('User is logged in', "https://" + extractDomain(window.location.href));
```

Comment:
Oh, it actually is just pulling domain. We're going to need to pull the origin (sentry server could be hosted on localhost, or on a weird port, etc, or not even https).